### DISCUSSION TITLE
logs in the enforce mode

### DISCUSSION THREAD

**Discussion by beroal** [roddhjav/apparmor.d#494] [General]
I understand why you require logs to be collected in the complain mode. However, switching a program to the complain mode defeats security. I create and fix profiles in the enforce mode. If I write

- what I saw in the AppArmor log and
- what I added to a profile until the log messages disappeared,

will you accept such a “log”?

**Comment by roddhjav** [comment]
That would only take longer for you... But on my side, it looks ok.
